include/sys/fs/zfs.h

```diff
@@ -1042,6 +1042,9 @@ typedef struct vdev_stat {
 	uint64_t	vs_trim_bytes_est;	/* total bytes to trim */
 	uint64_t	vs_trim_state;		/* vdev_trim_state_t */
 	uint64_t	vs_trim_action_time;	/* time_t */
+	uint64_t	vs_configured_ashift;   /* TLV vdev_ashift */
+	uint64_t	vs_logical_ashift;	/* vdev_logical_ashift  */
+	uint64_t	vs_physical_ashift;	/* vdev_physical_ashift */
```

Comment:
We need to be careful here because this structure is passed between user and kernel space.  Do we need to add this at all, based on the rest of the PR I don't see any user space consumers of these fields.